### ISSUE TITLE
Meet NAN when train icdar_2013 dataset only

### ISSUE THREAD

**Issue by ZhengMengbin** [MhLiao/MaskTextSpotterV3#32]
I use the configs/pretrain/seg_rec_poly_fuse_feature.yaml to train icdar_2013 dataset, and use default config（Use two 1080ti GPUs）:
![image](https://user-images.githubusercontent.com/26650043/97465527-1a6eaf80-197d-11eb-9cb8-636ac4258c30.png)
But, I meet the Nan loss. Is the learning rate too high, can you give some advice? The loss is follows:
![image](https://user-images.githubusercontent.com/26650043/97466049-a2ed5000-197d-11eb-93a7-5ebbc216122d.png)

**Comment by czh0001** [comment]
hello，I met the same problem when I trained on icdar2015 and I lowered the learning rate and solve the problem. And here I have another question to consult you: when we train on icdar2013 in this program, do we refer to the task 2:text segmentation? And what is the meaning of this 9 coordinates? I only know in icdar2015, this 8 values means the coordinate of the groundtruth region.
![1](https://user-images.githubusercontent.com/38340373/97947527-83fd1c80-1dc8-11eb-930f-e45687d78049.png)
![2](https://user-images.githubusercontent.com/38340373/97947528-8495b300-1dc8-11eb-944d-d3bc9734f4b3.png)

**Comment by czh0001** [comment]
hello，I met the same problem when I trained on icdar2015 and I lowered the learning rate and solve the problem. And here I have another question to consult you: when we train on icdar2013 in this program, do we refer to the task 2:text segmentation? And what is the meaning of this 9 coordinates? I only know in icdar2015, this 8 values means the coordinate of the groundtruth region.
![1](https://user-images.githubusercontent.com/38340373/97947615-ca527b80-1dc8-11eb-9a2b-f4fed93a2b1b.png)
![2](https://user-images.githubusercontent.com/38340373/97947627-d0e0f300-1dc8-11eb-8ba6-10c6b769abf0.png)

**Comment by czh0001** [comment]
hello，I met the same problem when I trained on icdar2015 and I lowered the learning rate and solve the problem.

**Comment by czh0001** [comment]
hello，I met the same problem when I trained on icdar2015 and I lowered the learning rate and solved the problem.

****

**Comment by MhLiao** [comment]
Some possible solutions:
(1) use smaller LR since your batch size is smaller
(2) pre-train the model with SynthText
(3) train the SPN alone first, and then train the whole network.

**Comment by samanthawyf** [comment]
@@ZhengMengbin hello, I met the same problem when training with my own dataset. Have you solved the problem and can you share any experience? @MhLiao
